port_ocean/core/ocean_types.py

```diff
@@ -30,6 +37,7 @@ class EntitySelectorDiff(NamedTuple):
 class CalculationResult(NamedTuple):
     entity_selector_diff: EntitySelectorDiff
     errors: list[Exception]
+    misconfigured: dict[str, str] = {}
```

Comment:
also https://www.pullrequest.com/blog/python-pitfalls-the-perils-of-using-lists-and-dicts-as-default-arguments/

Reply:
perhaps you can use defaultdict here